keyboards/jj50/keymaps/archetype/keymap.c

```diff
@@ -32,71 +32,139 @@ enum {
   TD_BSPC_DEL = 0,
   TD_LCTL_LBRC = 1,
   TD_LALT_RBRC = 2,
+  TD_ESC_LAYER = 3,
 };
 
 
 void left_brackets(qk_tap_dance_state_t *state, void *user_data) {
     if (state->count == 1) {
-      if (state->interrupted || !state->pressed) {
-        register_mods(MOD_LSFT);
+      if (!state->pressed) {
+        register_code(KC_LSFT);
```

Comment:
Just a heads up, you can convert this and the next line into just:
```c
register_code16(S(KC_9));
```
Or even:
```c
register_code16(KC_LEFT_PAREN);
```

And it will output the shifted character.

Reply:
thanks for that! I fixed it